pouta_blueprints/utils.py

```diff
@@ -121,3 +132,47 @@ def parse_port_range(port_range):
             raise ValueError('Port range invalid')
     else:
         raise ValueError('No port range found')
+
+
+def get_full_blueprint_config(blueprint):
+
```

Comment:
code is quite clear on what happens but the word "full" in function name is ambiguous. A quick docstring would help with this.